addons/mrp/static/src/components/bom_overview_line/mrp_bom_overview_line.js

```diff
@@ -170,9 +175,19 @@ export class BomOverviewLine extends Component {
     }
 
     get statusBackgroundClass() {
-        if(this.data.index == "0") {
-            return "text-bg-info";
+        if(this.data.availability_state == "available" || !this.data.level && this.data.availability_state == "estimated" && this.data.producible_qty
+                                                           && this.data.components_available){
```

Comment:
```suggestion
        if (
            this.data.availability_state == "available" ||
            (!this.data.level &&
                this.data.availability_state == "estimated" &&
                this.data.producible_qty &&
                this.data.components_available)
        ) {
```
Formatting. Won't suggest the other one, but *strongly* encourage you to [enable the linter](https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#tooling), it takes less than two minutes and keeps the code readable.